### DISCUSSION TITLE
PANEL: Latina Women polyglots in software development!

### DISCUSSION THREAD

**Discussion by David-Lazaro-Fernandez** [gittogethers/octogatosconf22#81] [Talks in Spanish]
### Moderator 
> Marian Villa
### Panelists 
> Bea Gandica, Fernanda Ochoa, Vane Marely, Manu Castrillón 

**From west to east (UTC -5,4,3,2) inspiring Latinas in Tech Industry.**

**Comment by carlosrivas2021** [comment]
Que increíble panel, sin duda son unas genias!

Una frase que me gusta mucho:
Lo imposible solo cuesta un poco mas!

**Reply by marianvilla** [reply]
Qué lindo David por estar y compartirnos esta frase <3 GRACIAS

**Comment by NoemiFN** [comment]
Pura mujer increíble!!! Motivan muchísimo. Yo hablo inglés, francés, español y algo de alemán, espero poco a poco lograr ser tan exitosa como todas ellas.

**Reply by marianvilla** [reply]
Wowww!!! Luego te veremos a ti en el panel. Un abrazo <3
